android/src/com/mapswithme/util/sharing/SharingHelper.java

```diff
@@ -215,12 +215,14 @@ private void updateItem(SharingTarget item)
 
   public static void shareBookmarksCategory(Activity context, int id)
   {
-    String name = BookmarkManager.INSTANCE.nativeSaveToKmzFile(id, MwmApplication.get().getTempPath());
+    String path = MwmApplication.get().getTempPath();
+    String name = BookmarkManager.INSTANCE.nativeSaveToKmzFile(id, path);
     if (name == null)
       return;
 
-    shareOutside(new LocalFileShareable(context, name + ".kmz", "application/vnd.google-earth.kmz")
-                     .setText(R.string.share_bookmarks_email_body)
+    shareOutside(new LocalFileShareable(context, path + name + ".kmz", "application/vnd.google-earth.kmz")
+                              // TODO fix translation for some languages, that doesn't contain holder for filename
```

Comment:
Уже да
https://jira.mail.ru/browse/MAPSME-811

> On Apr 12, 2016, at 2:14 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/util/sharing/SharingHelper.java <https://github.com/mapsme/omim/pull/2872#discussion_r59358544>:
> 
> >      if (name == null)
> >        return;
> >  
> > -    shareOutside(new LocalFileShareable(context, name + ".kmz", "application/vnd.google-earth.kmz")
> > -                     .setText(R.string.share_bookmarks_email_body)
> > +    shareOutside(new LocalFileShareable(context, path + name + ".kmz", "application/vnd.google-earth.kmz")
> > +                              // TODO fix translation for some languages, that doesn't contain holder for filename
> А есть таска?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2872/files/96fe261347de1686492a780544d838d0513f804b#r59358544>